Dockerfile

```diff
@@ -23,3 +23,20 @@ FROM release as legacy
 COPY entrypoint-legacy.sh /usr/bin/
 
 ENTRYPOINT ["/usr/bin/entrypoint-legacy.sh"]
+
+# Browser-enabled bundle
+FROM release as with-browser
+
+USER root
+
+COPY --from=release /usr/bin/k6 /usr/bin/k6
+RUN apk --no-cache add chromium-swiftshader
+
+USER k6
+
+ENV K6_BROWSER_ENABLED=true
+ENV CHROME_BIN=/usr/bin/chromium-browser
+ENV CHROME_PATH=/usr/lib/chromium/
+ENV CHROMIUM_FLAGS="--disable-software-rasterizer --disable-dev-shm-usage"
```

Comment:
What's the reason for setting these flags?

Can we use `ENV K6_BROWSER_HEADLESS=true` and rely on the default flags that the browser module works with?

Reply:
Good catch on the headless flag!

`--disable-dev-shm-usage` prevents Chrome from using `/dev/shm` that is [too small by default in Docker](https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#tips), and makes it use `/tmp` instead.
`--disable-software-rasterizer` turns off RAM-greedy GPU rasterizer, again due to limited resources available to the container.

Reply:
I've added a comment that explains what are these flags for.

Reply:
The browser module sets `disable-dev-shm-usage` when it launches a chrome instance. I've not witnessed any issues so far in an running container that would require `disable-software-rasterizer`, and the browser module doesn't use that flag.

My concern is that this flag could alter the behaviour of chrome somehow that could make it difficult to debug issues compared to when we run it inside/outside of a container without that flag.

If you still think it's worth adding, then could you work with `K6_BROWSER_ARGS` instead of `CHROMIUM_FLAGS`? I'm 100% sure that any new args are added to the launch process when working with `K6_BROWSER_ARGS`, but i'm not sure if flags defined in `CHROMIUM_FLAGS` are ignored or not when we launch chrome from the browser module.

Reply:
Makes sense, I'll move the args to the `K6_BROWSER_ARGS` to make sure we have control over them.

Reply:
Interesting, running it with `--` and space as a delimiter did not make any difference and my tests has passed successfully. I'm now leaning towards the initial suggestion from @ankur22 and remove these flags.